crates/cxx-qt-lib/src/types/qvector/mod.rs

```diff
@@ -330,3 +369,26 @@ impl_qvector_element!(u8, qvector_u8, "QVector_u8");
 impl_qvector_element!(u16, qvector_u16, "QVector_u16");
 impl_qvector_element!(u32, qvector_u32, "QVector_u32");
 impl_qvector_element!(u64, qvector_u64, "QVector_u64");
+
+#[cfg(test)]
+mod test {
+    use super::*;
+
+    #[test]
+    fn vec_from_qvec() {
+        let mut qvec = QVector::<u8>::default();
```

Comment:
we haven't been putting tests in `cxx-qt-lib` yet, as they create exciting linker errors that you just found :-) Instead they have been going into `tests/qt_types_standalone` and that also tests that they work with a CXX rather than a CXX-Qt bridge.

Reply:
What is also interesting is that macOS doesn't fail, although it appears the test is not in the exclude-regex ?